pkg/api/user.go

```diff
@@ -183,6 +194,16 @@ func (hs *HTTPServer) UpdateUserActiveOrg(c *models.ReqContext) response.Respons
 }
 
 func (hs *HTTPServer) handleUpdateUser(ctx context.Context, cmd user.UpdateUserCommand) response.Response {
+	// external user -> user data cannot be updated
+	isExternal, err := hs.isExternalUser(ctx, cmd.UserID)
+	if err != nil {
+		return response.Error(http.StatusInternalServerError, "Failed to validate User", err)
+	}
+
+	if isExternal {
+		return response.Error(http.StatusForbidden, "User info cannot be updated for external Users", nil)
+	}
+
```

Comment:
Hi @Jguer 
Is there any reason why we are preventing the external users from being updated?

Reply:
@Bujupah I have the exact same problem. I manage users with the terraform provider. Currently e.g. toggling the admin flag isn't possible (It work in the UI, though 🤔 )

Reply:
👋 Can you please open a new bug report explaining what problem you're running into? It's really hard to keep track of conversations when they're happening in already merged PRs 🙂